### ISSUE TITLE
Bug when using the Bootstrap’s datetimepicker in Safari

### ISSUE THREAD

**Issue by kevinnascimento** [Eonasdan/bootstrap-datetimepicker#1148]
There is a weird bug when using the Bootstrap’s datetimepicker in Safari (it happens only when February is shown). I haven’t found any issue to help me, is there any possible solution?

![screen shot 2015-06-26 at 10 15 50 am](https://cloud.githubusercontent.com/assets/5147127/8378335/00c2d624-1bef-11e5-8b51-fec9d2b3699b.png)

**Comment by Eonasdan** [comment]
There's a bug with wekit that causes this in certain TZs. I think this is fixed (from the picker) on Chrome, but I don't have Safari so I can't test this.

**Comment by gwynjudd** [comment]
Is this still outstanding? We have an issue that seems very similar:

Configure an iPad to use “Auckland, New Zealand” as the time zone in the General > Date & Time settings page.
Open ‘http://eonasdan.github.io/bootstrap-datetimepicker/’ in Safari on your iPad.
Drop down the picker from the ‘Minimum Setup’ input.
Under September 2015 you will see two 26th September entries, with the second being displayed in its own column.
Change to October 2015 and you will see that the dates do not update – the extra 26th September column still exists.  I suspect the dates are not being re-rendered for October.
Try selecting a date in October and then closing the picker.  The date will appear in the input control.  Try dropping the picker again – the picker will not drop down if the input control has a date in October 2015.
These problems also exist in September and October 2016, and presumably other years too.
The “Sydney , Australia” time zone also exhibits the problem although the 3rd October is shown twice in this case.

**Comment by gwynjudd** [comment]
I did a little testing, I can't replicate it in an iPad with iOS 9 or in the simulator running iOS 9, but I can in the simulator running 8.4

**Comment by camiloclc** [comment]
I'm having this exact same issue right now, only on Firefox on Windows 10 (not on Chrome or Edge or IE). At first I noticed it on a project I'm working on, but it's also happening on another project and on every single example which show the calendar on http://eonasdan.github.io/bootstrap-datetimepicker/. Every month of October for every year (like 2015, 2014, 2020, 1900, etc) is showing a duplicate of the third saturday, like on the image by @kevinnascimento.

**Comment by colinweight** [comment]
Similar for me, Melbourne timezone, it happens in Safari on a Mac with *first* week of October, no matter what year. 
![screen shot 2015-10-06 at 5 29 51 pm](https://cloud.githubusercontent.com/assets/5125043/10301628/ebf105f2-6c4f-11e5-834b-159b42530d5d.png)

**Comment by ricardocarrer** [comment]
same problem, but only after the summer schedule (rule of some states of Brazil).

![captura de tela 2015-10-20 as 15 22 48](https://cloud.githubusercontent.com/assets/8516820/10615587/7471d7e0-773f-11e5-937f-b0591bbc0e5d.png)

**Comment by lucygeneric** [comment]
Quite a few issues for me when swapping to/from DST.  BS3.datetimepicker should probably catch the errors but it's understandable that it isn't as Apple Webkit & moment.js are behaving most unpleasantly!

Safari Version 8.0.7 (10600.7.12)
```
> moment('Wed Sep 30 2015 23:00:00 GMT+1200').startOf('week').toString()
< "Sat Sep 26 2015 23:00:00 GMT+1200"
> moment('Wed Oct 14 2015 23:00:00 GMT+1200').startOf('week').toString()
< "Sun Oct 11 2015 00:00:00 GMT+1300"
```

Chrome Version 46.0.2490.71 (64-bit)
```
moment('Wed Sep 30 2015 23:00:00 GMT+1200').startOf('week').toString()
"Sun Sep 27 2015 00:00:00 GMT+1200"
moment('Wed Oct 14 2015 23:00:00 GMT+1200').startOf('week').toString()
"Sun Oct 11 2015 00:00:00 GMT+1300"
```

**Comment by mateomarin** [comment]
Did someone figure out a solution?

**Comment by ricardocarrer** [comment]
Yes. Thanks

Att,
Ricardo Carrer

**Comment by camiloclc** [comment]
How it looks on Firefox:

![clipboard02](https://cloud.githubusercontent.com/assets/1048942/19059694/438c9a34-89b8-11e6-9c23-8d641117220b.png)
![clipboard04](https://cloud.githubusercontent.com/assets/1048942/19059696/4693a45c-89b8-11e6-9b5c-18bc61a1deff.png)

**Comment by ricardocarrer** [comment]
I did not find solution, let like this.

Att,
Ricardo Carrer
